bin/crew

```diff
@@ -234,9 +234,13 @@ def generate_compatible
   Dir[CREW_PACKAGES_PATH + '*.rb'].each do |filename|
     pkgName = File.basename filename, '.rb'
     set_package pkgName, filename
+    puts "Checking #{pkgName} for compatibility.".orange if @opt_verbose
     if @pkg.compatibility.include? 'all' or @pkg.compatibility.include? ARCH
       #add to compatible packages
+      puts "Adding #{pkgName} to compatible packages.".lightgreen if @opt_verbose
       @device[:compatible_packages].push(name: @pkg.name)
+    else
+      puts "#{pkgName} not a compatible packages.".lightred if @opt_verbose
```

Comment:
`not a compatible package.` singular?

Reply:
This seems like a bit of an overkill to print these messages for every package.  We're talking over 1500 lines.

Reply:
I guess since it only happens with the verbose option, it's fine.

Reply:
Thanks!